src/Components/test/E2ETest/ServerExecutionTests/InteropReliabilityTests.cs

```diff
@@ -445,7 +445,7 @@ public async Task DispatchingEventsWithInvalidEventHandlerId()
             Assert.Contains(
                 logEvents,
                 e => e.eventIdName == "DispatchEventFailedToDispatchEvent" && e.logLevel == LogLevel.Debug &&
-                     e.exception is ArgumentException ae && ae.Message.Contains("There is no event handler with ID -1"));
+                     e.exception is ArgumentException ae && ae.Message.Contains("There is no event handler with ID 1"));
```

Comment:
I'm unsure why this is changing. Could you clarify?

Reply:
Oh wait I guess it's my fault after the switch to `ulong`, but I did run all the E2E tests locally. Any idea why it's failing for you now but didn't for me before?

Reply:
Likely because I put this change on the dependency update PR, because fixing the breaking changes from extensions in there was more cumbersome, when the PR for that was already signed off.

We didn't catch it because the PR got rebased and we don't run E2E tests on the CI :(